.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,12 +3,13 @@ about: "If you're sure it's reproducible and not just your machine: submit an is
 labels: bug
 issue_body: false
 inputs:
-  - type: description
+  - type: markdown
     attributes:
       value: Please note we will close your issue without comment if you do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
   - type: textarea
     attributes:
       label: "`brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output"
+    validations:
       required: true
   - type: checkboxes
     attributes:
```

Comment:
for some reason I can't click on the specific line to make the code suggestion but `choices` here should be changed to `options`

Reply:
Fixed, thanks @Oneill38!